python-tests.yml

```diff
@@ -0,0 +1,69 @@
+name: Python Tests
```

Comment:
why the duplicate file with the one in `.github/workflows`?  is this file being used?